tests/TestCase.php

```diff
@@ -81,4 +83,21 @@ function (
             );
         }
     }
+
+    protected static function getFixerByName(string $name): AbstractFixer
```

Comment:
We use it only in 3 test files. I wouldn't put this directly to master TestCase file.

Reply:
What is the number of usages to put a shared function in `TestCase` class? 3.5?

What would you suggest instead then?

Reply:
There are 2 traits for assertions there, are you sure you want to have a trait `GetFixerByNameTrait` or how would you name it?